src/hooks/analytics/segment-adapter.ts

```diff
@@ -49,14 +92,51 @@ export class SegmentAdapter {
     this.segment.track({
       anonymousId: this.id,
       event: options.command.replace(':', ' '),
+
+      context: await this.getSegmentEventContext(),
       properties: {
         ...pick(options.flags, ['platform', 'installer']),
-        command: options.command
+        command: options.command,
+        version: getProjectVersion()
```

Comment:
It seems we collection version twice.
Is it ok to remove it from this place?

Reply:
Yes,  Woopra ignore version from segmentEventContext. I will removed from here: https://github.com/che-incubator/chectl/blob/red_hat_tele/src/hooks/analytics/segment-adapter.ts#L134